tests/Feature/UserTest.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace Tests\Feature;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+
+use App\Church;
+use App\User;
+
+class UserTest extends TestCase
+{
+  use RefreshDatabase;
+
+  protected $response;
+  protected $church;
+  protected $user;
+	
+  protected function setUp(): void
+  {
+    parent::setUp();
+    
+    $this->church = factory(Church::class)->create();
+    $this->response = $this->post(route('users.store'), [
+      'id' => 1,
+      'name' => 'Test Name',
+      'email' => 'email@example.com',
+      'password' => 'password',
+      'church_id' => $this->church->id
+    ]);
+      
+    $this->user = factory(User::class)->create(['church_id' => $this->church->id]);
```

Comment:
It looks like we only use this second user in `test_church_has_multiple_users`, it might make sense to move this line to that test.